openfl/interface/workspace.py

```diff
@@ -108,34 +107,41 @@ def export_():
     except Exception:
         echo(f'Plan file "{planFile}" not found. No freeze performed.')
 
-    requirements_filename = 'requirements.txt'
-    workspace_reqs = _get_requirements_dict(requirements_filename)
-    prefix = getcwd()
-
-    export_requirements_filename = 'requirements.export.txt'
-    with open(export_requirements_filename, "w") as f:
-        check_call([executable, "-m", "pip", "freeze"], shell=False, stdout=f)
-    workspace_hash = _get_dir_hash(prefix)
-    origin_dict = _get_requirements_dict(
-        OPENFL_USERDIR / f'requirements.{workspace_hash}.txt')
-    current_dict = _get_requirements_dict(export_requirements_filename)
-    with open(export_requirements_filename, "w") as f:
-        for package, version in current_dict.items():
-            if (
-                package in workspace_reqs
-                or package not in origin_dict
-                or version != origin_dict[package]
-            ):
-                # we save only the difference between original workspace after
-                # 'fx create workspace' and current one.
-                echo(f'Writing {package}=={version} '
-                     f'to {requirements_filename}...')
-                f.write(f'{package}=={version}\n')
-            elif version is None:  # local dependency
-                warn(f'Could not generate requirements for {package}.'
-                     f' Consider installing it manually after workspace'
-                     f' import.')
-    echo(f'{export_requirements_filename} written.')
+    # requirements_filename = 'requirements.txt'
```

Comment:
Does the removal of this break the existing export mechanism for the legacy API? If not, this commented out code should be removed

Reply:
The new version works. Removed commented code